packages/api/src/elastic/pages.ts

```diff
@@ -28,14 +28,21 @@ import {
 } from './types'
 
 const appendQuery = (builder: ESBuilder, query: string): ESBuilder => {
-  const fields = ['title', 'content', 'author', 'description', 'siteName']
+  const fields = [
```

Comment:
One thing here is I think we should include the note field if possible. So we have scoped note search, but also do text search of the notes when someone does a query, since I think in most cases this content would be relevant to the user.

Reply:
If we include `note` field in the query, the `boost` score and `tie_breaker` might not work very well because `note` is actually stored as a nested field `highlights.annotation` and the `multi_match` query probably does not work with nested fields

Reply:
Let me try if it breaks the results

Reply:
It looks okay to add `note` field in the basic search: https://github.com/omnivore-app/omnivore/pull/2457#issuecomment-1617799805